packages/web-react/src/components/Collapse/useCollapseAriaProps.ts

```diff
@@ -0,0 +1,60 @@
+import { useRef, Ref, CSSProperties, MutableRefObject } from 'react';
```

Comment:
Please add a test for this if it is possible.

Reply:
Done